jbmc/src/java_bytecode/README.md

```diff
@@ -256,9 +256,72 @@ To be documented.
 
 To be documented.
 
-\section java-bytecode-lazy-methods-v1 Context Insensitive lazy methods (aka lazy methods v1)
-
-To be documented.
+\section loading-strategies Loading strategies
+
+Loading strategies are policies that determine what classes and method bodies
+ are loaded. On an inital pass, the symbols for all classes in all paths on the 
+ Java classpath are loaded. Eager loading is a policy that then loads method 
+ bodies for every one of these classes and for all the method bodies in these
```

Comment:
This sentence doesn't make sense to me... we load all the method bodies for all the method bodies...